include/vcpkg/tools.h

```diff
@@ -30,15 +31,9 @@ namespace vcpkg
         static const std::string IFW_REPOGEN = "ifw_repogen";
     }
 
-    enum class RequireExactVersions
-    {
-        YES,
-        NO,
-    };
-
     struct ToolCache
     {
-        virtual ~ToolCache() { }
+        virtual ~ToolCache();
```

Comment:
```suggestion
        virtual ~ToolCache() = default;
```
no reason to put this in the C++ file I think